hopr/hopr-lib/src/lib.rs

```diff
@@ -519,7 +519,10 @@ impl Hopr {
 
         let (tx_indexer_events, rx_indexer_events) = async_channel::unbounded::<SignificantChainEvent>();
 
-        let channel_graph = Arc::new(RwLock::new(ChannelGraph::new(me_onchain.public().to_address())));
+        let channel_graph = Arc::new(RwLock::new(ChannelGraph::new(
+            me_onchain.public().to_address(),
+            cfg.network_options.quality_avg_window_size as usize,
```

Comment:
Isn't it too high though? The auto-path needs a different function that everaging. It needs a function that reacts super slow for increasing quality, but is super fast on the decreasing quality.